### PR TITLE
TNT: Log explosions and player igniting tnt or gunpowder.

### PR DESCRIPTION
**Pull request by sofar**

### CODE REVIEW

**Comment by sofar** [comment]
Fixes #1520

**Comment by sofar** [comment]
example logs:
```
2017-01-17 15:17:09: ACTION[Server]: sofar ignites tnt:tnt at (1764,18,481)
2017-01-17 15:17:13: ACTION[Server]: A TNT explosion occurred at (1764,18,481) with radius 3

2017-01-17 15:17:29: ACTION[Server]: sofar ignites tnt:gunpowder at (1769,18,480)
2017-01-17 15:17:33: ACTION[Server]: A TNT explosion occurred at (1768,18,477) with radius 6

2017-01-17 15:20:45: ACTION[Server]: sofar uses fire:flint_and_steel, pointing at [node under=1774,19,487 above=1774,20,487]
2017-01-17 15:20:50: ACTION[Server]: A TNT explosion occurred at (1778,19,487) with radius 3
```

File: mods/tnt/init.lua [hunk 1]
```diff
@@ -411,6 +413,9 @@ minetest.register_node("tnt:gunpowder", {
 	on_punch = function(pos, node, puncher)
 		if puncher:get_wielded_item():get_name() == "default:torch" then
 			minetest.set_node(pos, {name = "tnt:gunpowder_burning"})
+			minetest.log("action", puncher:get_player_name() ..
+				" ignites " .. node.name .. " at " ..
```
Comment (paramat) [inline mods/tnt/init.lua]: No need for 'node.name' here since it's always gunpowder, this can be simpler code with no concatenation.

**Comment by paramat** [comment]
+1 if my line comment is addressed.
One request for your commit messages, there should be no period at the end, i have to remove these each time i merge one of your commits.

**Comment by sofar** [comment]
>+1 if my line comment is addressed.
>One request for your commit messages, there should be no period at the
>end, i have to remove these each time i merge one of your commits.

Is that a style rule? For most of my professional commits, we require full sentences, including period, so this would be awkward, to say the least, and even if I'd try to not do so, I'm fairly sure I'd fail quickly... :)

**Comment by paramat** [comment]
Hmm it's not present in any rules, i think it's an unofficial undocumented rule, looking through commit messages the first line almost never has a period at the end. I was told at some point to not use them. If you don't remember to no problem it's not a big issue.

**Comment by Zeno-** [comment]
Since I use a monospaced font can everyone please make sure there are two spaces after a fullstop?

**Comment by sofar** [comment]
All fixed up. Considering this is trivial, I'll merge myself.

**Comment by paramat** [comment]
:+1:

**Pull request merged**